### PR TITLE
Removes usage of admin Gamelab Gary.

### PR DESCRIPTION
**Pull request by ashercodeorg**
This PR is safe as we no longer require `admin` or `student_of_admin` status to access Game Lab. Further, this test would soon break (e.g., by PR#9155) as we prevent admins from accessing any levels.

Note that this removes the only instance of an admin that is created and persisted more than temporarily on `test`.

This PR (in effect) reverts #8222 and #8242.

### CODE REVIEW

**Comment by ashercodeorg** [comment]
islemaster: Not being familiar with eyes test, please assume that I don't know what I'm doing.

File: dashboard/test/ui/step_definitions/steps.rb [hunk 1]
```diff
@@ -672,42 +672,6 @@ def generate_user(name)
...
-
-  steps %Q{
-    When I type "#{email}" into "#user_login"
-    And I type "#{password}" into "#user_password"
-    And I click selector "input[type=submit][value='Sign in']"
-    Then I wait to see ".header_user"
-  }
-end
-
-And(/I display toast "([^"]*)"$/) do |message|
```
Comment (islemaster) [inline dashboard/test/ui/step_definitions/steps.rb:694]: :/ I know this doesn't get used anywhere else (yet) but I'm tempted to say leave it in - it's not specific to being an admin, and it's a nice way to call out particular steps in UI test recordings.

**Comment by islemaster** [comment]
LGTM.

I see that you made gamelab public again here: https://github.com/code-dot-org/code-dot-org/pull/9017  I'm surprised that I didn't hear about this.  Did you run this by Poorva?  We originally had it locked down for security reasons (however vague).

**Comment by ashercodeorg** [comment]
Opening Game Lab up was done at @poorvasingal's instruction. In response, we are (very soon) preventing admin users from accessing levels (of any type).

And I'll add another commit adding back the `I display toast` with a TODO to evaluate its usage status down the road.

**Comment by ashercodeorg** [comment]
PTAL.

**Comment by islemaster** [comment]
:+1: LGTM.  Thanks!

**Pull request merged**